frontend/pages/queries/details/QueryDetailsPage/QueryDetailsPage.tsx

```diff
@@ -248,12 +248,13 @@ const QueryDetailsPage = ({
   );
 
   const renderReport = () => {
-    const disabledCachingGlobally = true; // TODO: Update accordingly to config?.server_settings.query_reports_disabled
-    const discardDataEnabled = true; // TODO: Update accordingly to storedQuery?.discard_data
+    const disabledCachingGlobally =
```

Comment:
the default value for the global setting is `false` – are you sure we want default `true` here? It's just used for determining which `<NoResults />` message to display

Reply:
@RachelElysia